### DISCUSSION TITLE
First Post Test..

### DISCUSSION THREAD

**Discussion by NRTnarathip** [NRTnarathip/SMAPI-Android-1.6#1] [General]
im just test

**Comment by blackpeanut16** [comment]
hi are you releasing smapi?

**Reply by NRTnarathip** [reply]
yes

**Reply by blackpeanut16** [reply]
thank you very much.. i hope it sooner.......

**Reply by NRTnarathip** [reply]
https://github.com/NRTnarathip/SMAPILoader/wiki/How-To-Install-SMAPI

Done
